gemfiles/rack_3.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rack", "~> 3.0", git: "https://github.com/rack/rack.git"
+gem "rack-test", git: "https://github.com/rack/rack-test.git"
```

Comment:
Unless you have a good reason not to, I would drop the `git` option for rack-test.

Reply:
It doesn't seem to work without it.

Reply:
It seems this is needed here because we're out of date in the gemspec with regards to `rack-test`.

This PR should fix that https://github.com/rack/rack-attack/pull/587, so that we don't need this here.